src/bit-systems/delete-entity-system.ts

```diff
@@ -0,0 +1,49 @@
+import { defineQuery, exitQuery, hasComponent, removeEntity } from "bitecs";
+import { Vector3 } from "three";
+import { HubsWorld } from "../app";
+import { Deletable, HoveredRemoteLeft, HoveredRemoteRight } from "../bit-components";
+import { paths } from "../systems/userinput/paths";
+import { animate } from "../utils/animate";
+import { findAncestorEntity } from "../utils/bit-utils";
+import { coroutine } from "../utils/coroutine";
+import { easeOutQuadratic } from "../utils/easing";
+
+// TODO Move to coroutine.ts when it exists
+// TODO Figure out the appropriate type and use it everywhere
+export type Coroutine = Generator<Promise<void>, void, unknown>;
+
+const END_SCALE = new Vector3().setScalar(0.001);
+function* deleteEntity(world: HubsWorld, eid: number): Coroutine {
+  const obj = world.eid2obj.get(eid)!;
+  yield* animate([[obj.scale.clone(), END_SCALE]], 400, easeOutQuadratic, ([scale]: [Vector3]) => {
+    obj.scale.copy(scale);
+    obj.matrixNeedsUpdate = true;
+  });
+  removeEntity(world, eid);
+}
+
+const deletableQuery = defineQuery([Deletable]);
+const deletableExitQuery = exitQuery(deletableQuery);
+const hoveredRightQuery = defineQuery([HoveredRemoteRight]);
+const hoveredLeftQuery = defineQuery([HoveredRemoteLeft]);
+const coroutines = new Map();
+
+function deleteTheDeletableAncestor(world: HubsWorld, eid: number) {
+  const ancestor = findAncestorEntity(world, eid, (e: number) => hasComponent(world, Deletable, e));
```

Comment:
Hmm, what to call `Deletable`. `Removable` is equally bad.. `RemovableRoot`, `UserRemovable`? Not very good either..